noir-projects/aztec-nr/aztec/src/test/helpers/test_environment/test.nr

```diff
@@ -0,0 +1,173 @@
+use crate::test::{helpers::test_environment::TestEnvironment, mocks::mock_struct::MockStruct};
+
+use dep::std::mem::zeroed;
+
+global storage_slot: Field = 13;
+global value: MockStruct = MockStruct { a: 17, b: 42 };
+
+#[test(should_fail_with = "cannot be less than current timestamp")] // should be called 'next' timestamp probably
```

Comment:
`next` sounds good to me. We also refer to it randomly in TXE as `pending` but I like `next` more